administration/src/util/translations/Readme.md

```diff
@@ -0,0 +1,6 @@
+# Important Notice
+
+- Translation keys inside the `application` namespace should not be edited!!!
+- These keys are stored in the database and have to be resolvable.
+- If you change these keys, labels of old applications can not be resolved anymore.
+- Labels that can be changed because they are not stored in the database were put in other namespaces f.e. "applicationForms".
```

Comment:
:thought_balloon: just as an idea, because I'm pretty sure that if someone wants to change the lable, they won't look at the readme before doing so.
what if we add an json entry to the "application" like:
"_comment" : "Important! These labels can't be changed because ...."
it will be ignored by the app but serve as a visible comment

Reply:
thats a good idea, i will do that